src/bytes.rs

```diff
@@ -0,0 +1,13 @@
+use crate::{Decode, Encode, EncodeLike, Error, Input};
```

Comment:
Missing license header. (just copy from one of the other files).

And could you please also add at least some basic test?